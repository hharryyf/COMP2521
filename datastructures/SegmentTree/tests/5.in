10
9 8 12 4 5 1 5 25 3 -1
g 1 10
g 2 3
g 2 5
u 4 -10
g 3 7
g 1 10
