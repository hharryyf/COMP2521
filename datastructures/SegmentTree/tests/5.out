-1
8
4
-10
-10
